### ISSUE TITLE
[Bug]: Scroll bar shows, but not clickable

### ISSUE THREAD

**Issue by valdearg** [nextcloud/desktop#5536]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The scroll bar in the popout box will show, but it cannot be clicked on. This makes it difficult to scroll back to the top when lots of items are in the history.

![image](https://user-images.githubusercontent.com/2789033/226204099-db9d4a6d-5022-4920-a5a4-1f1629744b33.png)

### Steps to reproduce

1. Click on the NextCloud icon to bring up the popup menu.
2. Hover mouse over the right-hand side to show the scroll bar.
3. Click on the scrollbar icon and you should be able to scroll up and down.
4. You will find nothing happens. 
...

### Expected behavior

Nothing happens when clicking on the scroll bar/

### Which files are affected by this bug

NA

### Operating system

Windows

### Which version of the operating system you are running.

Windows 11

### Package

Distro package manager

### Nextcloud Server version

25.0.4

### Nextcloud Desktop Client version

3.7.4

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by JL102** [comment]
+1 on this. Made it completely impossible to scroll to the top of the UI without closing and reopening the app.

**Comment by JL102** [comment]
BTW: pressing Page Up / Page Down also doesn't work.

**Comment by fwcd** [comment]
Would be awesome if this could be fixed, the list unfortunately isn't really usable when having thousands of files to be synced. A small scroll-to-top button would probably fix this already.

**Comment by CarJem** [comment]
This is really making it a headache to resolve conflicts....

**Comment by q2apro** [comment]
I am a new Nextcloud user ... THIS is the first thing that came to my attention. Windows 10.

I just wanted to add a bug report myself.

<br>

Cannot use Scrollbar with Mouse, see screenrecording:

![2023-10-02 Nextcloud](https://github.com/nextcloud/desktop/assets/6764548/0d894a3a-731e-4220-9ddd-a124445809bb)

The scrollbar width is too small. The grabber cannot be touched.

1. Sync many files.
2. Open the desktop client with click in the system tray.
3. Try to move the scrollbar grabber.

---

Please fix as soon as possible. This is **essential**. Thank you! 🙏

**Comment by 9a3eedi** [comment]
I'd like to also add that if there's a lot of files being synced, the scrollbar (which you can't click anyway) becomes tiny to the point where you can't find it anymore. I'd suggest also adding a limiter to the number of items displayed in the list

**Comment by q2apro** [comment]
@Developers Please fix this. And when changing the code please fix the following too:

⚠️ 1. When opening the desktop client (Win 10) with clicking on the little tray icon, jump to the **first entry of the list**.

There is currently no way to reach the first entry, only scrolling an hour if the list is long. Or restarting the client (which is not helpful).

⚠️ 2. When clicking an entry in the desktop client list, please **open the folder on the PC**. 

Currently it opens the browser on the startpage, which is completely senseless.

Thank you for making Nextcloud better. 🙇

**Comment by 9a3eedi** [comment]
@q2apro in my opinion, these should be listed as separate bugs/issues. But I'm not a developer.

**Comment by JL102** [comment]
@9a3eedi / @q2apro 
Yes, I'm not a Nextcloud developer but generally you should report different issues in different, well, issues. But make sure to search the list of existing issues to make sure you are not reporting duplicates.

**Comment by q2apro** [comment]
@JL102 Feel free to report the separate issues. Thank you.

**Comment by NickHatBoecker** [comment]
Same bug on MacOS. Please fix this 🙏

**Comment by narFnarF** [comment]
Same issue on Linux Fedora, in the Gnome desktop environment.

![image](https://github.com/user-attachments/assets/33def7e7-57c2-40b2-aa35-df9e7f6186ae)
Nextcloud desktop client 3.13.0

Note that it is actually possible to grab the scrollbar if you grab it exactly on the edge on the left side. You need to be extra-precise, which is annoying and not intuitive.

![image](https://github.com/user-attachments/assets/bb97e477-fe14-4674-b8f7-c3255d26010b)

**Comment by JL102** [comment]
This appears to be fixed for me on the Windows client 3.14.1.20240927. However, it's worth noting that Page up / Page down still do nothing, since I noted that before. (though the lack of Page up / Page down is not a concern for me personally). Additionally, the update that fixed this seemed to introduce a somehow just as frustrating bug, #7269

**Comment by TomasTokaMrazek** [comment]
The bar is not even visible in dark mode.

**Comment by barisbasar1209** [comment]
@JL102 I agree that this problem seems to have been resolved since Windows client version **3.14.x** . I tried to replicate this issue and indeed my scroll bar worked just fine. 

@TomasTokaMrazek could you send a demonstration of what it looks like in your case. Since the original poster also used dark mode and there it was visible I am not sure what you mean. In my case somehow it does not even turn the widget dark, which is even weirder. Only the icons get either dark or completely white, which makes them almost invisible. 

@joshtrichards I saw this issue labeled as _good first issue_ but I suspect that it is already fixed. Are you the right person to decide this? Because in this case I would suggest closing the issue. All the other mentioned problems should probably be restated as separate issues, right?

Thanks.

**Comment by TomasTokaMrazek** [comment]
![Image](https://github.com/user-attachments/assets/e8373608-40ca-4dfd-b412-a0d7f056d68a)

It is visible only when the mouse is on the bar.

![Image](https://github.com/user-attachments/assets/6c651958-e4a1-4049-a51d-e83fa0b27821)

And when I scroll through mouse wheel, there is no animation and the position of items is always the same, so it's not even apparent, that scrolling is happening.

**Comment by barisbasar1209** [comment]
@mgallien Thanks for managing the PR! 
I wanted make sure the issue is also closed, so that new problems related to this topic won't be posted in here and get lost as a consquence. 

Could you close this issue?

**Comment by joshtrichards** [comment]
Fixed in  #8690
